5.1.i_have_no_vinger.py

```diff
@@ -0,0 +1,10 @@
+import datetime
+import random
+
+fDate = datetime.datetime.strptime(input("write a date (YYYY-MM-DD):"), '%Y-%m-%d')
+sDate = datetime.datetime.strptime(input("write a date (YYYY-MM-DD):"), '%Y-%m-%d')
+
+randDate = datetime.datetime.fromtimestamp(random.uniform(fDate.timestamp(), sDate.timestamp()))
+
+if (randDate.weekday() == 0):
```

Comment:
Expressions outside of a function are a bad habit. This code will be called every time the file is loaded and imported.   Read more here: https://www.geeksforgeeks.org/python-main-function/amp/